### PR TITLE
Update Beam version and code changes to meet current Dataflow requirements

### PR DESCRIPTION
**Pull request by nielm**
Update Beam version in pom.xml
Reformat and fix various code issues
Update run.sh to meet current dataflow requirements.

### CODE REVIEW

**Comment by nielm** [comment]
@googledrew

File: pom.xml [hunk 1]
```diff
@@ -36,7 +36,7 @@
```
[the remark this answers is not in the document]
Comment (googledrew) [reply pom.xml:36]: As an aside, both 2.36.0 and 2.37.0 resulted in errors.

[the remark this answers is not in the document]
Comment (nielm) [reply pom.xml:36]: I know (due to [BEAM-14005](https://issues.apache.org/jira/browse/BEAM-14005)) which is why I only updated to 2.35.0 
2.38 has now been released, so will update to that version

File: run.sh [hunk 2]
```diff
@@ -1,4 +1,19 @@
 #!/bin/bash
+#
+# Copyright 2019 Google LLC
```
[the remark this answers is not in the document]
Comment (nielm) [reply run.sh:3]: done

File: src/main/java/com/google/cloud/bqetl/BQETLNested.java [hunk 3]
```diff
@@ -1,12 +1,12 @@
 /*
  * Copyright 2019 Google LLC
```
[the remark this answers is not in the document]
Comment (nielm) [reply src/main/java/com/google/cloud/bqetl/BQETLNested.java:2]: done

File: src/main/java/com/google/cloud/bqetl/json/JSONReader.java [hunk 4]
```diff
@@ -1,12 +1,12 @@
 /*
  * Copyright 2019 Google LLC
```
[the remark this answers is not in the document]
Comment (nielm) [reply src/main/java/com/google/cloud/bqetl/json/JSONReader.java:2]: done

File: src/main/java/com/google/cloud/bqetl/mbdata/MusicBrainzDataObject.java [hunk 5]
```diff
@@ -1,12 +1,12 @@
 /*
  * Copyright 2019 Google LLC
```
[the remark this answers is not in the document]
Comment (nielm) [reply src/main/java/com/google/cloud/bqetl/mbdata/MusicBrainzDataObject.java:2]: done

File: src/main/java/com/google/cloud/bqetl/BQETLSimple.java [hunk 6]
```diff
@@ -139,25 +138,26 @@ private static TableSchema bqSchema() {
         .intField("artist_end_date_month")
         .intField("artist_end_date_day")
         .intField("artist_type")
-// [START schemaCodeChange]
-/*Switch these two lines when using mapping table for artist_area */
-//        .stringField("artist_area")
+        // [START schemaCodeChange]
+        // Uncomment this line, and comment out the following when using lookups for artist_area.
```
[the remark this answers is not in the document]
Comment (nielm) [reply src/main/java/com/google/cloud/bqetl/BQETLSimple.java:142]: Switched to using a command line flag to trigger lookups

File: src/main/java/com/google/cloud/bqetl/BQETLSimple.java [hunk 7]
```diff
@@ -28,107 +28,106 @@
...
+    /*
+     * load the line delimited JSON into keyed PCollections
+     */
     // [START loadArtistsWithLookups]
-    //PCollection<KV<Long,MusicBrainzDataObject>> artists = MusicBrainzTransforms.loadTable(p,"artist","id",
-    //        MusicBrainzTransforms.lookup("area", "id", "name", "area", "begin_area"),
-    //        MusicBrainzTransforms.lookup("gender","id","name","gender"));
-
-    PCollection<KV<Long, MusicBrainzDataObject>> artists = MusicBrainzTransforms.loadTable(p, "artist", "id");
+    // Uncomment this statement, and comment out the following to performing area/gender lookups
```
[the remark this answers is not in the document]
Comment (nielm) [reply src/main/java/com/google/cloud/bqetl/BQETLSimple.java:63]: Switched to using a command line flag to trigger lookups

**Review by googledrew — APPROVED** [review]
LGTM

**Pull request merged**